docs/docs/reference/docgen/nextjs/content/use-sign-in-sms-passwordless.mdx

```diff
@@ -0,0 +1,50 @@
+---
+# ⚠️ AUTO-GENERATED CONTENT. DO NOT EDIT THIS FILE DIRECTLY! ⚠️
+title: useSignInSmsPasswordless()
+sidebar_label: useSignInSmsPasswordless()
+slug: /reference/nextjs/use-sign-in-sms-passwordless
+description: Use the hook `useSignInSmsPasswordless` to sign in a user using a one-time password sent by SMS on a phone.
+custom_edit_url: https://github.com/nhost/nhost/edit/main/packages/react/src/useSignInSmsPasswordless.ts#L57
+---
+
+# `useSignInSmsPasswordless()`
+
+Use the hook `useSignInSmsPasswordless` to sign in a user using a one-time password sent by SMS on a phone.
+
+1. The `signInSmsPasswordless` action will send a one-time password to the given phone number.
+2. The client is then awaiting the OTP. `needsOtp` is set to true
+3. After the code is received by SMS, the client sends the code with `sendOtp`. On success, the client is authenticated, and `isSuccess` equals `true`.
```

Comment:
In 2. you're using "...is set to true". 
In 3. you're using "... equals `true`". 

Maybe we can use the language for both?